ami/db.go

```diff
@@ -27,10 +27,18 @@ func DBPut(client Client, actionID, family, key, val string) (Response, error) {
 
 // DBGet gets DB Entry.
 func DBGet(client Client, actionID, family, key string) (Response, error) {
-	return send(client, "DBGet", actionID, dbData{
+	data := dbData{
 		Family: family,
 		Key:    key,
-	})
+	}
+
+	responses, err := requestList(client, "DBGet", actionID, "DBGetResponse", "DBGetComplete", data)
+
+	if err != nil {
+		return nil, err
```

Comment:
It's better verify if the response is `len(responses)` greater then zero first, for example:
```go
switch {
  case err != nil :
     return nil, err
 case len(responses) == 0:
     return nil, errors.New("there is no db entries")
}
return responses[0], nil
```

Reply:
tks

Reply:
I correct one grammar mistake, the correct form is: `there are no db entries`, instead `there is...`

Reply:
done